packages/redux-window-message-thunk/README.md

```diff
@@ -0,0 +1,67 @@
+# `redux-window-message-thunk`
+
+A middleware that allows you to setup event handlers for window 'message' events.
+
+## Philosophy
+We like thunks and want to use them wherever possible.  We should be able to handle all errors that occur inside of executing thunks, both sync and async in a centralized place.  When we handle an error, the error handling function is just an action creator that may itself return another thunk.
+
+## Usage
+
+```bash
+npm install redux-window-message-thunk
+```
+
+Then in the place you're creating your redux store, add the middleware:
+
+```js
+import { createStore, applyMiddleware } from 'redux';
+import reduxWindowMessageThunk from 'redux-window-message-thunk';
+import myReducerFn from './myReducer.js';
+
+const messageCallbacks = {
+  'MODAL_CLOSE': (data) => {
+    const {someKey} = data
+    return {
+      type: 'HANDLE_MODAL_IN_PARENT',
+      payload: {someKey}
+    }
+  }
```

Comment:
Example shows a message type being mapped to an action, which is fine, but it given that the name of the middleware is `redux-window-message-thunk` should also probably show message being mapped to a thunk.

```
  `MODAL_OPEN' : (data) => {
      return (dispatch, getState) => { 
            const {someKey} = data;
            if (getState().isAwesome) {
              dispatch({ type: 'MODAL_OPEN_WHILE_AWESOME', someKey }); 
            }
      }
}
```

Reply:
Added an example of using a thunk below